sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusClientBuilder.java

```diff
@@ -208,13 +209,23 @@ public ServiceBusReceiverClientBuilder buildReceiverClientBuilder() {
      */
     void onClientClose() {
```

Comment:
Probably not urgent, but should we also add Autocloseableon receiver/sender builder because some time a user might just want to close the connection from builder in order to stop everything ?  We could decide about it later.

Reply:
This is a design decision, you can create an issue for it.. but it seems weird for a builder to implement autocloseable.